server/backend/api/app/crud/crud_user.py

```diff
@@ -246,9 +247,131 @@ def fetch_initial_requester_info(
         select(models.FamUser)
         .options(
             joinedload(models.FamUser.fam_access_control_privileges),
-            joinedload(models.FamUser.fam_user_terms_conditions)
+            joinedload(models.FamUser.fam_user_terms_conditions),
         )
         .filter(models.FamUser.cognito_user_id == cognito_user_id)
     )
     user = db.scalars(q_stm).unique().one_or_none()
     return user
+
+
+def update_user_info_from_idim_source(
+    db: Session, use_pagination: bool, page: int, per_page: int
+) -> schemas.FamUserUpdateResponse:
+    """
+    Go through each user record in the database, update the user information to match the record in IDIM web service
+    """
+    # get a requester from the database
+    requester = (
+        db.query(models.FamUser)
+        .filter(models.FamUser.user_name == "CMENG")
```

Comment:
we need to setup a requester (valid IDIR user) in order to call IDIM web service, I just put my name here.

Reply:
thanks Ian, I added a method in the config.py